src/services/LoggerService.ts

```diff
@@ -40,13 +40,15 @@ import { ConnectionService } from './ConnectionService';
 import { DockerService } from './DockerService';
 import { IService } from './IService';
 import { ServiceLocator } from './ServiceLocator';
+import { ResourceCreationState } from '../state/ResourceCreationState';
 
 
 export enum LogBoard {
     CONSENSUS = 'CONSENSUS',
     MIRROR = 'MIRROR',
     RELAY = 'RELAY',
-    ACCOUNT = 'ACCOUNT'
+    ACCOUNT = 'ACCOUNT',
+    RESOURCE = 'RESOURCE'
```

Comment:
We have a practice of adding commas in the end of newly added lines as well. In this way when we add a new line the change will appear only there